action.yml

```
@@ -14,6 +14,10 @@ inputs:
   upload-chunk-size:
     description: 'The chunk size used to split up large files during upload, in bytes'
     required: false
+  enableCrossOsArchive:
+    description: 'An optional boolean enabled to save and restore cache on windows which could be restored and saved on any platform'
```

Suggestion for description: Optional boolean when enabled, allows windows to restore caches created on other platforms.

---

```suggestion
    description: 'An optional boolean when enabled, allows to save and restore caches on windows restored and saved on other platforms'
```
This sound better? We need to have save and restore both

---

```suggestion
    description: 'An optional boolean when enabled, allows windows runners to save/restore caches that can be restored/saved respectively on other platforms'
```
True, we need save/restore both. How about this?

---

Sounds good

---

@bishal-pdMSFT Does this sound better or this https://github.com/actions/cache/pull/1056#discussion_r1061408475 ?

---

Looks good. The word "respectively" does not seem right fit as we are using `/`. But I will agree my english is not so good.

---

We can remove respectively, I'm also not too sure if this would be grammatically correct.